web-services/deploy/application/pom.xml

```diff
@@ -124,12 +124,14 @@
             <version>${project.version}</version>
             <type>ejb</type>
         </dependency>
+        <!--
         <dependency>
             <groupId>gov.nsa.datawave.webservices</groupId>
             <artifactId>datawave-ws-query-websocket</artifactId>
             <version>${project.version}</version>
             <type>war</type>
         </dependency>
+	-->
```

Comment:
Why removed? 
(This comment applies to all the other related pom and java code updates here as well)

Reply:
Made a comment in this pom about the websocket project being unused code that after the jackson/wildfly upgrade, needs re-configuration in order to compile and deploy.  Didn't want to just delete the code in case we want to actually stand it up and use it in the near future.